src/app/components/ScrollablePromo/index.jsx

```diff
@@ -38,12 +37,12 @@ const PromoWrapper = styled.div`
   }
 `;
 
-const LabelComponent = styled(SectionLabel)`
-  margin: 0;
-  padding: 0;
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    margin: 0 0 ${GEL_SPACING};
-  }
+const LabelComponent = styled.strong`
+  display: block;
+  ${({ script }) => script && getBodyCopy(script)}
+  ${({ service }) => service && getSerifBold(service)}
+  font-size: 20px;
```

Comment:
line height as well?

Reply:
Sorry, @greenc05 I still need to update this. I think UX wants me to use the same font-size, line-height and colour we used [here](https://5d28eb3fe163f6002046d6fa-adcwqshcry.chromatic.com/?path=/story/containers-front-page-section--pidgin)

Reply:
Cool. Really, ok. It will also need to change per breakpoint as usual, and also have a relative font-size